### DISCUSSION TITLE
Midi Thru

### DISCUSSION THREAD

**Discussion by qwamp** [probonopd/MiniDexed#619] [Q&A] [answered]
Hello!
Please tell me, is it possible to specify two sources (umidi1, ttyS1) for midi thru at the same time?
It would be great to use different devices with midi thru output in ttyS1 without changing the configuration every time.

**Comment by diyelectromusic** [comment, marked as answer]
I believe we only pick up the first MIDIThru line in minidexed.ini, so the answer would be no I'm afraid.
MiniDexed certainly doesn't do any kind of intelligent merging, if that is what you're asking :)
